Data/SNC_Data_Convention.md

```diff
@@ -23,7 +22,7 @@ Encoding format: UTF-8
 [
 	{
 		"stress_ratio": double, // Stress ratio (R) [-]
-		"reliability_level": int, // Reliability level (rsql) [-]
+		"confidence_interval": double, // confidence bounds, usually 5, 95% (rsql) [-]
 		"a": double, // S-N Curve parameter [-]
 		"b": double, // S-N Curve parameter [1/MPa]
 		"lrsq": double, // [?]
```

Comment:
aren't you missing fp here ??

Reply:
you're right, fp is missing, I forget it because it's not used in all methods (not mendatory)